CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+## Contributing
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+We accept pull requests for bug fixes and small features where the approach will be simple and straightforward. We'd also love to hear about ideas for new features as issues.
+
+We track issues on our project board [here](https://github.com/OffchainLabs/arbitrum-token-bridge/issues).
+
+Please do:
+
+- Check existing issues to verify that the bug or feature request has not already been submitted.
+- Open an issue if things aren't working as expected.
+- Open an issue to propose a significant change.
+- Open a pull request to fix a bug.
+- If you're contributing for the first time here, try and look for any issues labelled [good first issue](https://github.com/OffchainLabs/arbitrum-token-bridge/contribute).
```

Comment:
Do we want them to assign themselves? Otherwise we might have issues with multiple people working on the same issue

Reply:
External contributors don't have the access to assign the PR to anyone. 
But this brings up a good point - we should also mention that we will revoke the assignments to stale PRs or the ones without any progress till X days. Need to confirm this number with the team.

Reply:
external contributors won't have the permission to assign themselves

Reply:
if we go with the assign flow, it'd make sense to impose a time limit onto the assignment, e.g.
https://github.com/OffchainLabs/arbitrum-token-bridge/issues/681#issuecomment-1506873144

Reply:
Added this.
> Please avoid: 
> - Keeping issues assigned to yourself for long: If you've been assigned an issue, please be mindful that if there's no PR/activity in a week's time, you'll be unassigned.